reference/content-types/smart_content.rst

```diff
@@ -0,0 +1,71 @@
+Smart content
+=============
+
+Description
+-----------
+
+Shows a list of pages, which depends on a configureable filter. You can define
+which which site's children should be included, what tags the filtered pages
```

Comment:
`You can include branches of the content sites tree and tagged content, the results can be sorted and limited` .. or something. At least it should not say `which which` :)